ci/travis/build-docker-images.py

```diff
@@ -247,9 +247,11 @@ def get_new_tag(old_tag, new_tag):
 
         for arch_tag in ["-cpu", "-gpu", ""]:
             full_arch_tag = f"nightly{arch_tag}"
-            # Tag and push rayproject/<image>:nightly<arch_tag>
-            docker_push(full_image, full_arch_tag)
+            if not _release_build():
```

Comment:
Can you toss a comment about "do not tag nightly on release branch because it may lag behind the actual latest nightly build."